### ISSUE TITLE
Incorrect reporting of an issue from import/no-named-as-default

### ISSUE THREAD

**Issue by amiramix** [import-js/eslint-plugin-import#2894]
@testing-library exports both, default and named export using the same name, i.e.:

```
export { userEvent as default, userEvent } from './setup/index.js';
```

We then import it with

```
import userEvent from '@testing-library/user-event'
```

Which eslint incorrectly reports as error:

```
 error  Using exported name 'userEvent' as identifier for default export
```

Can the rule be modified to accept a list of named exports to ignore?

**Comment by VigneshElangovan52** [comment]
Facing a similar issue when we try to upgrade testing-library/user-event to v14.4.3. Were you able to find any workaround or so? @amiramix 
Any updates or addresses from the team would be very helpful. Thanks!

**Comment by ljharb** [comment]
This is a bug that should be fixed; adding an ignore list would just be papering over the problem.

**Comment by akwodkiewicz** [comment]
Duplicate of https://github.com/import-js/eslint-plugin-import/issues/1594
